.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -34,4 +34,4 @@ Steps to reproduce the behavior:
 
 <!-- Check the box below with an `x` if you are planning to create a PR for this. -->
 
-- [ ] I'm planning to make a pull-request
\ No newline at end of file
+- [ ] I'm planning to make a pull-request
```

Comment:
What were changed in this line?

Reply:
to be honest; I don't see it either 😕

Reply:
Let's hope we won't face strange diffs in the future with these plugins.